### DISCUSSION TITLE
Can not get bridge to connect to the plugin anymore (Offline or on celllular backup?)

### DISCUSSION THREAD

**Discussion by zeple101** [dgreif/ring#1248] [Q&A]
I have restarted HB, uninstalled and re-installed the plugin multiple times. No matter what I do I can't seem to get it working anymore. Everthing was working perfectly until one day it just, didn't. Kepe getting error message :Ring beams_bridge_v1 (xxxxxxxxxx) is offline or on cellular backup.:

Can anyone tell me how to fix this ?

**Comment by dgreif** [comment]
This error indicates that you have a beams bridge (eg for Ring Smart Lighting) registered on your account, but that bridge is not currently plugged in or connected to the internet. You can either reconnect that bridge, or remove it from your account to get past this error

**Reply by zeple101** [reply]
Fixed my issue, thanks again for the help !!

**Comment by zeple101** [comment]
Ohhh you’re right ! I do have my lighting bridge unplugged ! I’ll give that
a shot ! Thanks so much for your help !
